### ISSUE TITLE
Investigate Why Queries Against Federated Sources Don't Work

### ISSUE THREAD

**Issue by willwill96** [connexta/revelio#318]
```
ApolloError: GraphQL error: invalid json response body at https://localhost:8993/direct?_=1586371080173 reason: Unexpected end of JSON input
    at new ApolloError (webpack-internal:///./node_modules/apollo-client/bundle.esm.js:76:28)
    at eval (webpack-internal:///./node_modules/apollo-client/bundle.esm.js:1258:40)
    at eval (webpack-internal:///./node_modules/apollo-client/bundle.esm.js:1570:25)
    at Set.forEach (<anonymous>)
    at eval (webpack-internal:///./node_modules/apollo-client/bundle.esm.js:1568:32)
    at Map.forEach (<anonymous>)
    at QueryManager.broadcastQueries (webpack-internal:///./node_modules/apollo-client/bundle.esm.js:1566:22)
    at eval (webpack-internal:///./node_modules/apollo-client/bundle.esm.js:1657:27)
    at Object.next (webpack-internal:///./node_modules/zen-observable/lib/Observable.js:308:23)
    at notifySubscription (webpack-internal:///./node_modules/zen-observable/lib/Observable.js:130:18) {
  graphQLErrors: [
    FetchError: invalid json response body at https://localhost:8993/direct?_=1586371080173 reason: Unexpected end of JSON input
        at eval (webpack-internal:///./node_modules/isomorphic-fetch/node_modules/node-fetch/lib/body.js:48:31)
        at runMicrotasks (<anonymous>)
        at processTicksAndRejections (internal/process/task_queues.js:94:5) {
      message: 'invalid json response body at https://localhost:8993/direct?_=1586371080173 reason: Unexpected end of JSON input',
      locations: [],
      path: [Array]
    }
  ],
  networkError: null,
  message: 'GraphQL error: invalid json response body at https://localhost:8993/direct?_=1586371080173 reason: Unexpected end of JSON input',
  extraInfo: undefined
}
```

![image](https://user-images.githubusercontent.com/10997562/78823488-04c27a00-799a-11ea-822f-9c338dcf17c8.png)

**Comment by mazarag2** [comment]
We should change the wording of this issue I'm not seeing an issue with queries against ALL federated sources but only ODS-ElasticSearch and ScheduledNotifcationService at the moment
![federated query](https://user-images.githubusercontent.com/16808210/79778873-4a623980-82ee-11ea-9b41-f083d8e5abb3.png)

**Comment by gjvera** [comment]
Fixed by #346
